src/Home.js

```diff
@@ -1,159 +1,182 @@
-import React from 'react'
+import React, { useState } from 'react'
 import RatingSystem from './RatingSystem'
 import styled from 'styled-components/macro'
 
 export default function Home() {
+  const [questions, setQuestion] = useState([
+    {
+      id: '1',
```

Comment:
Du kannst die gesamte Liste als JSON auslagern und dann importieren

Reply:
Guck dafür gerne mal in mein Projekt rein, da hab ich eine JSON importiert in der Loot.js :)

Reply:
das mache ich danke :)